packages/suite/src/utils/wallet/transactionUtils.ts

```diff
@@ -495,7 +495,11 @@ const numberSearchFilter = (
     amount: BigNumber,
     operator: typeof searchOperators[number],
 ) => {
-    const targetAmount = getTargetAmount(transaction.targets[0], transaction);
+    const targetAmount =
+        transaction.targets[0] !== undefined
+            ? getTargetAmount(transaction.targets[0], transaction)
```

Comment:
now I realised, does it mean that you can't search a tx by the amount in, for example, its 3rd output? 
Eg. when you sent btc tx to multiple recipients: 1st will receive 0.1 btc, 2nd 0.2 btc, 3rd, 0.3b tc. Will I be able to find such transaction by typing "0.3"? It most likely won't affect many users, but would be nice to handle properly.

Anyway, I would merge this as it is since it should fix the issue, and for the rest we can create an issue for now. What do you think?

Reply:
aand it is merged

Reply:
That's a good point. Can't remember the exact reason it was done this way but I guess it's worth testing/investigating. If anyone from @trezor/qa could check it out and open an issue if relevant, that would be awesome :black_heart:

Reply:
i was able to reproduce it, i will send issue in the morning